README.md

```diff
@@ -702,6 +702,7 @@ Table of Contents
    * [kanbantool.com](https://kanbantool.com/) — Kanban board-based project management. Free, paid plans with more options
    * [kanrails.com](https://kanrails.com/) — Kanban board-based project management. Free for 3 collaborators, 2 projects and 5 tracks. Paid plans available for unlimited collaborators, projects and tracks.
    * [LeanBoard](https://www.leanboard.io) — Collaborative whiteboard with sticky notes for your GitHub issues (Useful for Example Mapping and other techniques)
+   * [MeuScrum](https://www.meuscrum.com) - Free online scrum tool with kanban board
```

Comment:
Perhaps, it's better to use https://www.meuscrum.com/en Otherwise it opens in non-English localization by default.

Reply:
weird, pretty sure when I checked it it was english default, that sounds good yeah, feel like a fix PR?

Reply:
I've just tested it again in a private windows in Firefox and it opens on Portuguese.

Reply:
I've submitted https://github.com/ripienaar/free-for-dev/pull/1454

Reply:
There are 3 localization options: 
https://www.meuscrum.com/en (english version)
https://www.meuscrum.com/pt (brazilian portuguese version)
https://www.meuscrum.com/es (spanish version)

https://www.meuscrum.com shows pt-br version by default.